Extend database check

infastra opened the pull request:
By starting the application it will be create the var-folder, so that the check will be false on every time and the script doesn't initialize the session tables. For fixing this the constructor takes a closer view if the sqlite-file exists. If not it knows that it is a new Sqlite-database.

infastra committed:
Extend database check

By starting the application it will be create the var-folder, so that the check will be false on every time and the script doesn't initialize the session tables. For fixing this the constructor takes a closer view if the sqlite-file exists. If not it knows that it is a new Sqlite-database.

drieschel commented on src/Session/SqliteSessionHandler.php:
```diff
@@ -53,8 +53,14 @@ public function __construct(string $databaseDir)
         $this->lifetime = (int)ini_get('session.gc_maxlifetime');
 
+        $dbLocation = sprintf('%s/connector.s3db', $databaseDir);
+        if (!file_exists($dbLocation))
+        {
+          $isNew = true;
+        }
```
```suggestion
   $isNew = !file_exists($dbLocation);
```

drieschel reviewed (changes requested):
Thanks for the PR. 

The `$isNew = true;` in line 51 is not necessary anymore and can get removed.

PS: Also `$isNew = false;` in line 46 can get removed.

infastra committed:
Update src/Session/SqliteSessionHandler.php

That's right! In this way you could remove line 50/51 ```isNew = true;``` as well.

Co-authored-by: Immanuel <drieschel@yahoo.de>

infastra committed:
Set variable $isNew is unnecessary
